src/components/atoms/Navbar/styles.tsx

```diff
@@ -1,76 +1,102 @@
 import styled from "styled-components";
 
-import { minDevice } from "../../../assets/styles/breakpoints";
-import { activeLink } from "../../../assets/styles/colors";
 import Wrapper from "../Wrapper";
 
-const NavbarDiv = styled(Wrapper)`
-  height: 8rem;
+import { minDevice } from "../../../assets/styles/breakpoints";
+import { activeLink, white } from "../../../assets/styles/colors";
+
+const HeaderWrapper = styled.div`
   justify-content: space-between;
   align-items: center;
+  position: relative;
+  z-index: 11;
+  display: flex;
+`;
+
+const NavbarWrapper = styled(Wrapper)`
+  height: 8rem;
+  width: 100%;
+  position: relative;
+  z-index: 12;
```

Comment:
I think this isn't needed.

Reply:
I've removed height and position, but width and z-index are necessary here.

Reply:
Can we have a quick call? I've removed it locally and was working well for me, maybe I missed something ;)